src/TaylorBot.Net/Core/src/TaylorBot.Net.Core/User/UserExtensions.cs

```diff
@@ -8,5 +8,14 @@ public static string GetAvatarUrlOrDefault(this IUser user, ImageFormat format =
         {
             return user.GetAvatarUrl(format, size) ?? user.GetDefaultAvatarUrl();
         }
+
+        public static string GetGuildAvatarUrlOrDefault(this IUser user, ImageFormat format = ImageFormat.Auto, ushort size = 128)
+        {
+            return user is IGuildUser guildUser ?
```

Comment:
Nit: we should consolidate the 2 checks like `user is IGuildUser guildUser && guildUser.GuildAvatarId != null ?`